feedthefox/base/templates/jinja2/flatpage.html

```diff
@@ -5,8 +5,23 @@
 {% block content %}
   <div class="row">
     <div class="col-sm-3">
-      <img src="{{ static('img/foxfooding.png') }}" alt="Foxfooding"
-           class="img-responsive">
+      {% block flatpage_image %}
+        <img src="{{ static('img/foxfooding.png') }}" alt="Foxfooding"
+             class="img-responsive">
+      {% endblock %}
+      {% block contribute_flatpages %}
+        <div class="contribute-flatpages">
+          <h3>Contribute</h3>
+          <ul>
+            <li><a href="{{ url('porting') }}">Porting Guide</a></li>
```

Comment:
Instead of manually iterating over our ``flatpages`` can't we just loop over dynamically?